src/binary-reader-ir.cc

```diff
@@ -1619,6 +1619,9 @@ Result BinaryReaderIR::OnReloc(RelocType type,
   if (options_.features.sandbox_enabled()) {
     return Result::Ok;
   }
+  if (index == kInvalidIndex) {
```

Comment:
maybe add a comment why we are doing this.. Ideally also check that this is indeed a debug relocation.

Reply:
I added a comment, including a link to the code that Sam showed us where these indices are generated.
I changed things so we tolerate -1 and also -2.
Checking the section name might be valuable, but it's a bit messy and we need to change the OnReloc interface to plumb the name through.  So I decided to not do this for now.  (This is only in the no-sandbox branch anyway.)